Subject: [PATCH] HTC_PASIC3 should depend on ARCH_PXA

drivers/mfd/htc-pasic3.c:22:31: error: asm/arch/pxa-regs.h: No such file or directory

or we should just drop the offending #include.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/mfd/htc-pasic3.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/mfd/htc-pasic3.c
+++ b/drivers/mfd/htc-pasic3.c
@@ -19,7 +19,6 @@
 #include <linux/interrupt.h>
 #include <linux/mfd/htc-pasic3.h>
 
-#include <asm/arch/pxa-regs.h>
 
 struct pasic3_data {
 	void __iomem *mapping;
